pkg/apis/kubevirt/validation/worker.go

```diff
@@ -19,38 +19,61 @@ import (
 
 	apiskubevirt "github.com/gardener/gardener-extension-provider-kubevirt/pkg/apis/kubevirt"
 
-	corev1 "k8s.io/api/core/v1"
+	gardencorev1 "github.com/gardener/gardener/pkg/apis/core"
+	k8scorev1 "k8s.io/api/core/v1"
```

Comment:
Please use `gardenercore` and `corev1` as aliases - `gardencorev1` is incorrect (there is no version), and `corev1` should not be changed for consistency with the rest of the codebase.
```suggestion
	gardenercore "github.com/gardener/gardener/pkg/apis/core"
	corev1 "k8s.io/api/core/v1"
```